docs/Using-Fleet/FAQ.md

```diff
@@ -31,13 +31,13 @@
 - [How often is the software inventory updated?](#how-often-is-the-software-inventory-updated)
 - [Can I group results from multiple hosts?](#can-i-group-results-from-multiple-hosts)
 
-## What do I need to do to switch from Kolide Fleet to FleetDM Fleet?
+## Upgrading Fleet
 
-The upgrade from kolide/fleet to fleetdm/fleet works the same as any minor version upgrade has in the past.
+To migrate to Fleet from Kolide Fleet, please follow the steps outlined in the Upgrading Fleet section of the documentation.
 
 Minor version upgrades in Kolide Fleet often included database migrations and the recommendation to back up the database before migrating. The same goes for the new Fleet.
 
-To migrate from `kolide/fleet` to the new Fleet, please follow the steps outlined in the [Upgrading Fleet section](../Deploying/Upgrading-Fleet.md) of the documentation.
+To migrate from Kolide Fleet to the new Fleet, please follow the steps outlined in the [Upgrading Fleet section](../Deploying/Upgrading-Fleet.md) of the documentation.
```

Comment:
```suggestion
```
This seems weird to have the same information in the same section. See my suggestions.